concepticondata/conceptlists/Wittmann-1973-200.tsv-metadata.json

```diff
@@ -0,0 +1,66 @@
+{
+    "@context": [
+        "http://www.w3.org/ns/csvw",
+        {
+            "@language": "en"
+        }
+    ],
+    "dialect": {
+        "encoding": "utf-8-sig",
+        "delimiter": "\t",
+        "skipBlankRows": true
+    },
+    "tables": [
+        {
+            "tableSchema": {
+                "columns": [
+                    {
+                        "datatype": {
+                            "base": "string",
+                            "format": "[a-zA-Z]+\\-[0-9]{4}\\-[0-9]+[a-z]?\\-[0-9]+[a-z]?$"
+                        },
+                        "name": "ID"
+                    },
+                    {
+                        "datatype": {
+                            "base": "string",
+                            "format": "[0-9\\.]+([a-z\\\u2013]+)?$"
+                        },
+                        "name": "NUMBER"
+                    },
+                    {
+                        "datatype": {
+                            "base": "integer",
+                            "minimum": 1
+                        },
+                        "name": "CONCEPTICON_ID"
+                    },
+                    {
+                        "datatype": "string",
+                        "name": "CONCEPTICON_GLOSS"
+                    },
+                    {
+                        "datatype": "string",
+                        "name": "ENGLISH"
+                    },
+                    {
+                        "datatype": "string",
+                        "name": "MAURITIAN"
+                    },
+                    {
+                        "datatype": "string",
+                        "name": "HAITIAN"
+                    },
+                    {
+                        "datatype": "string",
```

Comment:
@CarolinHu, you can be even a bit more explicit by adding the glottocodes in these cases, e.g., for Portuguese, you could write:

```
                    {
                        "datatype": "string",
                        "propertyUrl": "https://glottolog.org/resource/languoid/id/port1283",
                        "name": "PORTUGUESE"
                    },
```

We have not been doing this consequently, but it may be helpful for completeness.